updateParamGroups.iLogicVb

```diff
@@ -65,81 +83,80 @@ Public Class updateParamGroup
 	
 		Select Case BeforeOrAfter
 			Case kBefore
-				'MsgBox("kBefore")
 
```

Comment:
should probably have an Exit Sub here, right? I know at least in other languages cases roll over into one another without breaks, and if you're including an Else just to exit sub there, I'm not sure there's a reason not to include it here